plugins/install.go

```diff
@@ -215,6 +215,11 @@ func installLocalPlugin(pluginSourceURI, pluginLocalPth string) (Plugin, error)
 		}
 	}
 
+	err = os.RemoveAll(tmpPluginDir)
```

Comment:
What about wrapping the tmp dir removal with a defer right after the dir creating, so that it gets cleanup up when there is an error?

Reply:
Good idea, fixed in latest commit